site-src/concepts/cluster-profile-api.md

```diff
@@ -0,0 +1,57 @@
+# ClusterProfile API Overview
+
+This document provides an overview of the [ClusterProfile API](https://github.com/kubernetes-sigs/cluster-inventory-api?tab=readme-ov-file#cluster-profile-api).
+
+![Alt](../images/cluster-profile-api.png "ClusterProfile API")
+
+A Cluster Profile is essentially an individual member of the Cluster Inventory that details the properties and status of a cluster.
+This API proposes a universal, standardized interface that defines how cluster information should be presented
+and interacted with across different platforms and implementations.
+
+You can read more details about the API in the [KEP-4322](https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/4322-cluster-inventory/README.md).
+
+## Terminology
+
+- **Cluster Inventory**: A conceptual term referring to a collection of clusters.
+
+- **Cluster Manager**: An entity that creates the ClusterProfile API object per member cluster,
+  and keeps their status up-to-date. Each cluster manager MUST be identified with a unique name.  
+  Each ClusterProfile resource SHOULD be managed by only one cluster manager. A cluster manager SHOULD
+  have sufficient permission to access the member cluster to fetch the information so it can update the status
+  of the ClusterProfile API resource.
+
+- **ClusterProfile API Consumer**: the person running the cluster managers
+  or the person developing extensions for cluster managers for the purpose of
+  workload distribution, operation management etc.
+
+## API Example

```

Comment:
Could we point to the CRD definition? I think it's in sig-mc too.
@mikeshng would know the location.